README.md

```diff
@@ -11,7 +11,7 @@
 Grin is an in-progress implementation of the Mimblewimble protocol. Many characteristics are still undefined but the following constitutes a first set of choices:
 
   * Clean and minimal implementation, and aiming to stay as such.
-  * Follows the Mimblewimble protocol, which provides great anonymity and scaling characteristics.
+  * Follows the Mimblewimble protocol, which provides confidential transactions and scaling advantages.
```

Comment:
CT won't probably say much to most people, I would add something like `(hidden amounts)` behind.

Reply:
Fixed.